src/screens/Exchange/ExchangeOffers.js

```diff
@@ -550,10 +502,7 @@ class ExchangeOffers extends React.Component<Props, State> {
       || !allowanceSet
       || (isShapeShift && !shapeshiftAccessToken);
 
-    const isFiat = isFiatProvider(offerProvider);
-
-    const disableFiatExchange = isFiat && (minOrMaxNeeded || !!offerRestricted);
-    const disableOffer = disableNonFiatExchange || disableFiatExchange;
+    const disableOffer = disableNonFiatExchange;
```

Comment:
We can remove the `disableOffer` const now, just use `disableNonFiatExchange ` below